paymentsheet/src/test/java/com/stripe/android/paymentsheet/VerticalModePaymentSheetActivityTest.kt

```diff
@@ -301,6 +307,75 @@ internal class VerticalModePaymentSheetActivityTest {
         composeTestRule.onNodeWithText("Gimme money!").assertExists()
     }
 
+    @Test
+    fun `Saved payment method mandates work correctly`() = runTest(
+        primaryButtonLabel = "Gimme money!",
+        customer = PaymentSheet.CustomerConfiguration(id = "cus_1", ephemeralKeySecret = "ek_test"),
+        networkSetup = {
+            setupElementsSessionsResponse(lpms = listOf("card", "us_bank_account"))
+            setupV1PaymentMethodsResponse(usBankAccount1)
+            setupV1PaymentMethodsResponse(card1, card2)
```

Comment:
What does it mean to call `setupV1PaymentMethodsResponse` twice? I would've expected it would be called once with all three of usBankAccount1, card1, and card2

Reply:
This is just the way the stripe API works. If card, us_bank_account, and sepa_debit are all supported (the only 3 we support as saved payment methods), we'll call GET /v1/payment_methods 3 times 😬 

It's inefficient. But at least they happen in parallel.

Reply:
ah ok gotcha, thanks!